### DISCUSSION TITLE
An Inquiry on Github gists

### DISCUSSION THREAD

**Discussion by rezzcode** [community/community#84067] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I have been writing GitHub gists for some time and was wondering if there is a way to add clickable links on the gist texts?

**Comment by megamiii** [comment, marked as answer]
To add clickable links in your GitHub gists, make sure to save or rename your gist files with a `.md` extension, which denotes Markdown format. Then, use Markdown syntax to create hyperlinks. For example, write `[Link description here](http://url_here)` to create a clickable link. Here's how it looks in practice: `[Google](https://www.google.com)`. When viewed, [Google](https://www.google.com) will appear as a clickable hyperlink leading to https://www.google.com.

**Reply by rezzcode** [reply]
mmmh! that never come to my mind , thanks for the info.
